src/Notification/Notification.less

```diff
@@ -68,10 +70,33 @@
 	.moon-body-text-spacing {
 		margin: 0;
 	}
-
+	.moon-button {
+		min-width: 180px;
+		margin-left: 9px;	/* gap between buttons should be 9px per notification spec */
+		margin-right: 0px;
+		.enyo-locale-right-to-left & {
+			margin-left: 0px;
+			margin-right: 9px;
+		}
+	}
+	.moon-hspacing {
+		& > :first-child,
+		& > :first-child.moon-contextual-popup-decorator :first-child {
```

Comment:
Do we have any use case that is using contextual popup decorator on dialog popup?